virttest/qemu_vm.py

```diff
@@ -3669,6 +3670,15 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
             if offline is True:
                 self.monitor.cmd("stop")
 
+            if migrate_capabilities:
+                for key, value in migrate_capabilities.items():
+                    state = value == "on"
+                    self.monitor.set_migrate_capability(state, key)
+                    s = self.monitor.get_migrate_capability(key)
+                    if s != state:
+                        raise exceptions.TestError("Failed to get "
+                                                   "value of %s." % key)
```

Comment:
Has same question with @suqinguang.   exception looks very strange. better to:
"Migration capability '%s' should be '%s' , but it's '%s'" % ( value, s and "on" or "off")